ccpp/data/GFS_typedefs.meta

```diff
@@ -620,6 +634,27 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[dust12m_in]
+  standard_name = fengsha_dust12m_input
+  long_name = fengsha dust input
+  units = various
+  dimensions = (horizontal_dimension,12,5)
+  type = real
+  kind = kind_phys
+[emi_in]
+  standard_name = anthropogenic_background_input
+  long_name = anthropogenic background input
+  units = various
+  dimensions = (horizontal_dimension,1)
+  type = real
+  kind = kind_phys
+[smoke_GBBEPx]
+  standard_name = emission_smoke_GBBEPx
+  long_name = emission fire GBBEPx
+  units = various
+  dimensions = (horizontal_dimension,24,3)
+  type = real
+  kind = kind_phys
```

Comment:
You're missing the `active=` lines for these and many other variables. The non-smoke configurations will fail unless you have `active=` lines for variables that are not always allocated.